meet-data/spf-archive/0436/lifters.csv

```diff
@@ -0,0 +1,40 @@
+WeightClassKg,Division,Name,Age,BodyweightKg,State,BestSquatKg,BestBenchKg,BestDeadliftKg,TotalKg,Sex,Equipment,Event,Place
+82.5,Teenage (18-19),John Garrison,19,78.02,TN,204.12,124.74,226.8,555.65,M,Wraps,SBD,1
+67.5,Juniors,Kristian Bales,22,66.22,TN,97.52,45.36,131.54,274.42,F,Wraps,SBD,1
+82.5,Juniors,Sarah Knox,22,76.2,TN,117.93,54.43,147.42,319.78,F,Wraps,SBD,1
+110,Juniors,Franklin Snyder,20,107.95,TN,272.16,204.12,308.44,784.71,M,Wraps,SBD,1
+110,Juniors,Seth Phillips,21,108.86,TN,272.16,204.12,256.28,732.55,M,Wraps,SBD,2
+67.5,Open,Brittany Buckner,,63.96,TN,99.79,58.97,127.01,285.76,F,Wraps,SBD,1
+82.5,Open,Kathryn Sullivan,25,80.29,TN,120.2,65.77,129.27,315.25,F,Wraps,SBD,1
+100,Open,Rhonda Swaney,40,97.98,TN,147.42,77.11,158.76,383.29,F,Wraps,SBD,1
+90+,Sub-masters,Samantha Coleman,35,163.75,TN,242.67,158.76,235.87,637.3,F,Wraps,SBD,1
+90,Open,Dustin Reed,,88.9,TN,301.64,151.95,263.08,716.68,M,Wraps,SBD,1
+100,Open,James Pinet,26,98.88,GA,244.94,172.37,260.82,678.12,M,Wraps,SBD,1
+110,Open,Justin Bethune,,106.14,GA,344.73,204.12,322.05,870.9,M,Wraps,SBD,1
+110,Open,Kyle Slone,26,105.69,KY,226.8,151.95,265.35,644.1,M,Wraps,SBD,2
+117.5,Open,Justin Jones,31,106.59,TN,229.06,188.24,238.14,655.44,M,Wraps,SBD,2
+117.5,Open,Cameron Gordon,30,117.48,TN,229.06,156.49,278.96,664.51,M,Wraps,SBD,1
+125,Open,Michael Bales,29,122.47,TN,288.03,226.8,299.37,814.2,M,Wraps,SBD,1
+140,Open,Kyle Coleman,,131.54,TN,301.64,158.76,299.37,759.77,M,Wraps,SBD,1
+100,Masters (40-44),John Greeves,42,108.86,GA,192.78,149.69,235.87,578.33,M,Wraps,SBD,1
+117.5,Masters (40-44),Chris Moorse,41,117.48,TN,272.16,172.37,235.87,680.39,M,Wraps,SBD,1
+82.5,Masters (45-49),David Russell,45,81.19,TN,217.72,151.95,226.8,596.47,M,Wraps,SBD,1
+90,Masters (45-49),Larry Berry,49,89.36,TN,224.53,165.56,226.8,616.89,M,Wraps,SBD,1
+82.5,Masters (55-59),Arthur McCann,58,88,GA,122.47,99.79,204.12,426.38,M,Wraps,SBD,1
+82.5,Masters (45-49),David Russell,45,81.65,TN,217.72,,,217.72,M,Wraps,SBD,2
```

Comment:
David Russell was Squat-only here, so the Event should be "S". I'm surprised this passed tests! I gotta make the tests better...